### PR TITLE
feat: Points are aligned now

### LINKED ISSUE #4668: Points aren't in correct alignment!
### Description

Points aren't in correct alignment [here](https://64b5a6d647333e5a58a5b133--layer5.netlify.app/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures).
It would look good if 'application' and 'gateways' align with other points [here](https://64b5a6d647333e5a58a5b133--layer5.netlify.app/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition).
It would look good if 'Be' would be align with 'production' [here](https://64b5a6d647333e5a58a5b133--layer5.netlify.app/learn/service-mesh-books/istio-up-and-running).
### Expected Behavior

They should be aligned.

### Screenshots

image

image

image

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by abhijeetgauravm**
**Description**
Points are aligned now
This PR fixes #4668 

**Notes for Reviewers**
Points are aligned now
image

'application' and 'gateways' align with other points now
image

'Be'  align with 'production' now
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e6683bde62052441f5ccf53d2a964b52d209b873 at: https://64c8edacad29fb709ecbc835--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d2ff85f76ba13e4dbd424836e22ab4d22c2a5a04 at: https://64c8f2d79328880075654c20--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8660a29d2dd0f7c9c9da34a12da26816e92dc348 at: https://64c9184f6a38201d423798f4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ea1b49a66bd09533aa59f31e88a1b4669bfc60f6 at: https://64ca099b51987e72489eed91--layer5.netlify.app

File: src/collections/service-mesh-books/the-enterprise-path-to-service-mesh-architectures/index.mdx [hunk 1]
```diff
@@ -27,16 +27,18 @@ As someone interested in modern software design, you have heard of service mesh
...
 
 <div>
+<ul>
   <li> What is a service mesh and why do I need one? </li>
-    <li className="mr-l-15"> What are the different service meshes and how do they contrast?</li>
-    <li className="mr-l-15"> Where do services meshes layer in with other technologies?</li>
-  <li>When and why should I adopt a service mesh?</li>
-    <li className="mr-l-15">What are popular deployment models and why?</li>
+  <li> What are the different service meshes and how do they contrast?</li>
+  <li> Where do services meshes layer in with other technologies?</li>
```
Comment (Mohith234) [inline src/collections/service-mesh-books/the-enterprise-path-to-service-mesh-architectures/index.mdx:36]:
```suggestion
  <li> Where do service meshes layer in with other technologies?</li>
```
- I believe this is a valid change to be made

**Comment by l5io** [comment]
🚀 Preview for commit a4b0585f855e8b4d542c6ab556a250a874fab55f at: https://64cb4a55462f6f3e056fb636--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0ac1075baad8446661698f2dcb2559d4c773c7e1 at: https://64cd24ed759c682969030509--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
hey @abhijeetgauravm 


 Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by abhijeetgauravm** [comment]
ok @Ghat0tkach !

**Review by thisiskaransgit — APPROVED** [review]
@abhijeetgauravm Looks good to merge, thank you for the quick fix :tada:

**Comment by l5io** [comment]
🚀 Preview for commit da9defc02901240a2f0769c7f4ff38abfee047ff at: https://64d37f9456c1675aaea57638--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 051e010ccd3c99c954455de14fe3ac518c1070a3 at: https://64d385206bb6905dca4d356c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7402109bc48e1bdaf0408a2409d99fe5c3403595 at: https://64d3900aff81cd640ec1a0f6--layer5.netlify.app

**Pull request closed without merging**